app/src/main/java/com/eveningoutpost/dexdrip/Models/BloodTest.java

```diff
@@ -124,7 +124,7 @@ private BloodTestMessage toMessageNative() {
     private static final long CLOSEST_READING_MS = 30000; // 30 seconds
 
     public static BloodTest create(long timestamp_ms, double mgdl, String source) {
```

Comment:
Probably you made the fix in master, and I didn't pull it down yet, but made it my local repo.